docker/reference/normalize.go

```diff
@@ -89,16 +88,21 @@ func ParseDockerRef(ref string) (Named, error) {
 // needs to be already validated before.
 func splitDockerDomain(name string) (domain, remainder string) {
 	i := strings.IndexRune(name, '/')
-	if i == -1 || (!strings.ContainsAny(name[:i], ".:") && name[:i] != "localhost") {
+	if i == -1 || (!strings.ContainsAny(name[:i], ".:") && name[:i] != "localhost" && strings.ToLower(name[:i]) == name[:i]) {
```

Comment:
It’s not obvious we want this.

Reply:
Do you think we have tests to reveal that we want this?  If not I am fine with just removing the change.